### PR TITLE
Hide info on person page if they don't have a current role

### PR DESCRIPTION
**Pull request by tommyp**
This PR makes changes to the `person#show` page along with the presenter in order to hide certain information if the person doesn't currently hold any role. This includes hiding the image, truncating the biography at the end of the first paragraph and hiding the link to `Role` from the left hand nav. [Ticket](https://trello.com/c/bLzBWF8F/132-people-without-current-roles-should-have-limited-biographies).

Commits:
- hide image and link to Role in nav
- truncate biography at the end of the first paragraph for people without a current role
- modify an existing test as it was creating a Person with a previously held role but wasn't testing anything specific to this
- testing the changes to the person page with a feature test

Screenshot of the page of a person with no current role:

screen shot 2015-03-26 at 16 29 57

### CODE REVIEW

File: app/presenters/person_presenter.rb [hunk 1]
```diff
@@ -42,7 +42,13 @@ def speeches
   end
 
   def biography
-    context.govspeak_to_html model.biography
+    if in_current_role?
+      context.govspeak_to_html model.biography
+    else
+      # If a person doesn't currently hold any roles, we use a truncated
+      # version of the biography which ends after the first paragraph
+      context.govspeak_to_html model.biography.split(/\n/).first if model.biography
```
Comment (tekin) [inline app/presenters/person_presenter.rb]: I think this needs to be a bit safer, either by splitting the source markdown on newlines, or by actually selecting the first `<p>` tag from the generated HTML. Splitting HTML on whitespace does not guarantee we'll always get the first paragraph as there is no guarantee that the generated HTML doesn't contain whitespace characters elsewhere in the markup.

File: features/people.feature [hunk 2]
```diff
@@ -14,7 +14,7 @@ Scenario: Viewing all people
   And I should see that "Fred Bloggs" is listed under "b"
 
 Scenario: Viewing the person page for a minister
-  Given "Benjamin Disraeli" is a minister with a history
+  Given a person called "Benjamin Disraeli"
```
Comment (tekin) [inline features/people.feature]: The person this step creates will not have any roles, which means the scenario title is no inaccurate (they're no longer a minister).

File: features/people.feature [hunk 3]
```diff
@@ -65,4 +65,9 @@ Scenario: Images are virus-checked before publication
...
   Then the image will be quarantined for virus checking
   When the image has been virus-checked
-  Then the virus checked image will be available for viewing
\ No newline at end of file
+  Then the virus checked image will be available for viewing
+
+Scenario: Viewing a person that previously had a role
+  Given "Dale Cooper" is a minister with a history
+  When I visit the person page for "Dale Cooper"
+  Then I should see past information about the person "Dale Cooper"
```
Comment (tekin) [inline features/people.feature]: Maybe this step would more accurately describe the feature if it read something like:

```
Then I should see a truncated biography of the person
```

Comment (tommyp) [inline features/people.feature]: Does it matter that this step is actually testing more than just the truncated biography?

Comment (tekin) [inline features/people.feature]: No, I don't think so. Or maybe `Then I should see a limited biography of the person` is more appropriate. That might better encapsulate the lack of image, etc`.

Comment (tommyp) [inline features/people.feature]: Would `limited information` not make more sense?

Comment (tekin) [inline features/people.feature]: It would! :+1:

**Comment by tommyp** [comment]
Amended the commits based on @tekin's comments.

**Comment by tekin** [comment]
Good to go one build is green.

**Comment by tommyp** [comment]
- [x] product review from @bmwelby so I'm merging this.

**Pull request merged**